tests/Oryx.BuildImage.Tests/CommandTests/OryxCommandTest.cs

```diff
@@ -89,7 +89,7 @@ public void BuildImage_Build_UsesCwd_WhenNoSourceDirGiven()
             RunAsserts(
                 () =>
                 {
-                    Assert.Contains("Error: Could not detect", result.StdErr);
```

Comment:
Just confirming: with the new command line library, it's not writing an `Error` prefix?

Reply:
i think the new console doesn't provide Error or Warning tag anymore even using `console.Error.WriteLine`

Reply:
https://github.com/microsoft/Oryx/pull/1816/files#r1129817709